sssd_test_framework/hosts/base.py

```diff
@@ -296,3 +296,64 @@ def __init__(self, *args, **kwargs) -> None:
 
         self.fs: LinuxFileSystem = LinuxFileSystem(self)
         self.svc: SystemdServices = SystemdServices(self)
+        self._os_release: dict = {}
+        self._distro_name: str | None = None
+        self._distro_major: int | None = None
+        self._distro_minor: int | None = None
+
+    def _distro_information(self):
+        """
+        Pulls distro information from a host from /ets/os-release
+        """
+        self.logger.info(f"Detecting distro information on {self.hostname}")
+        os_release = self.ssh.run("cat /etc/os_release", log_level=SSHLog.Error).stdout_lines
```

Comment:
`/etc/os-release` not `/etc/os_release` right?

Reply:
fixed.

Reply:
I did not propagate local changes from debugging. :-)